src/quo2/components/settings/data_item/content/subtitle.cljs

```diff
@@ -0,0 +1,36 @@
+(ns quo2.components.settings.data-item.content.subtitle
+  (:require [react-native.core :as rn]
+            [quo2.components.settings.data-item.style :as style]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.foundations.resources :as resources]
+            [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [quo2.components.icon :as icons]))
+
+(defn view
+  [theme size description icon icon-color blur? subtitle]
+  (let [yellow-color [:yellow (if (= theme :dark) 60 50)]]
```

Comment:
@mmilad75 is this color always yellow? I believe this should be a prop `customization-color` like we have in most other quo2 components.
Ideally we use like

```
(defn view
  [theme size description icon icon-color blur? subtitle customization-color]
  (let [background-color (colors/theme-colors
   (colors/custom-colors customization 50)
   (colors/custom-colors customization 60)
theme)
```

  however here, you should (afaik) just be passing this prop directly to account-avatar -> i.e 
  
  ```
[account-avatar/view
                    {:customization-color customization-color
  ```

Reply:
you can check this doc about it here: https://www.notion.so/handling-custom-colors-d4060bc87cfb4c268d19a77f0711d3a1

Reply:
Done